canonical-web-and-design/ubuntu.com
# Update to site restricted custom search

## Done

Update all u.com search to site restricted search which avoids daily query limits.

## QA

- Check https://ubuntu-com-11424.demos.haus/tutorials?q=packer
- Check https://ubuntu-com-11424.demos.haus/search?q=ubuntu
- Check https://ubuntu-com-11424.demos.haus/server/docs/search?q=ubuntu
- Check https://ubuntu-com-11424.demos.haus/ceph/docs/search?q=ubuntu
- Check https://ubuntu-com-11424.demos.haus/core/docs/search?q=ubuntu
- Check https://ubuntu-com-11424.demos.haus/openstack/docs/search?q=ubuntu
- Check https://ubuntu-com-11424.demos.haus/security/livepatch/docs/search?q=ubuntu
- Check https://ubuntu-com-11424.demos.haus/security/certifications/docs/search?q=ubuntu

Make sure you find results

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
